.gitignore

```diff
@@ -7,3 +7,4 @@
 /dist/
 /docs/_build/
 /example/.venv/
+venv
```

Comment:
Don't add your personal requirements to the project's `.gitignore` file

```suggestion
```

Do set up a global git ignore file: https://adamj.eu/tech/2022/11/06/git-set-up-a-global-ignore-file/

But also you shouldn't need to create a `venv` dir locally - use `tox` to run the tests like `tox -e py311`.